ckanext/cnra_schema/templates/base.html

```diff
@@ -0,0 +1,6 @@
+{% ckan_extends %}
+
+{% block styles %}
+    {{ super() }}
+    <link rel="stylesheet" href="/css/additional_metadata_form.css?v=2" />
```

Comment:
Why was the css file moved here?

Reply:
I noticed that css files were being defined in this block and believed that it should have all the files in one area. If this is not the case, I will move it back to the original location.

Reply:
The css files imported in the base.html template is used everywhere on the site.

The additional_metadata_form.css file should only be moved here if it's used anywhere else in the site. Otherwise you add unnecessary overhead when loading a page that doesn't need it.

Reply:
That makes sense, I will move the css file to the original place.